Gemfile.lock

```diff
@@ -145,7 +145,7 @@ GEM
     factory_bot (6.3.0)
       activesupport (>= 5.0.0)
     ffi (1.15.5)
-    gds-api-adapters (90.0.0)
+    gds-api-adapters (91.0.0)
```

Comment:
Not blocker: We'd normally put gem updates in a separate commit to increase readability and reduce noise in the commit.

Reply:
Good shout – I can see Dependabot has already created #3153, I'll let you merge that and will rebase this PR!